src/framework/handlers/script.js

```diff
@@ -120,6 +145,38 @@ class ScriptHandler {
 
         head.appendChild(element);
     }
+
+    _loadModule(url, callback) {
+
+        // if we're in the browser, we need to use the full URL
+        const baseUrl = platform.browser ? window.location.origin : import.meta.url;
```

Comment:
Can an es5 version of the engine load module scripts? If so then `import.meta.url` would fail, right?

Reply:
(Even though that'll only ever be used in node).

Reply:
The target is [marginally higher](https://github.com/playcanvas/engine/blob/020e5f3b762e48cf6759d8430e85dabf36cf9d8b/utils/rollup-es5-options.mjs#L22-L24) than ES5 now, but babel should transpile this according to the preset.env. See [here](https://babeljs.io/docs/babel-plugin-syntax-import-meta)

Reply:
That sounds good, just double check it does and we're not missing some settings please.